app/views/tags/edit.html.erb

```diff
@@ -21,7 +21,7 @@
   <div class="slanty-accent"></div>
   <div class="tag-or-query-header-container tag-edit-tag">
     <h1> Editing:
-      <a href="/t/<%= @tag.name %>" style="color: <%= @tag.text_color_hex %>;text-decoration: underline;"><%= @tag.name %></a>
+      <a href="/t/<%= URL.tag @tag %>" style="color: <%= @tag.text_color_hex %>;text-decoration: underline;"><%= @tag.name %></a>
```

Comment:
forgot to highlight the bug :D this is going to hardcode `/t/` twice

Reply:
Oooh, fixing now